src/core/Akka/Event/LogFilter.cs

```diff
@@ -225,7 +225,7 @@ private EmptyLogFilterEvaluator() : base(Array.Empty<LogFilterBase>())
 
         public override bool ShouldTryKeepMessage(LogEvent evt, out string expandedLogMessage)
         {
-            expandedLogMessage = evt.Message.ToString()!;
```

Comment:
This is the fix for most use cases, when users don't have any log filtering enabled.